extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/AccessType.scala

```diff
@@ -69,7 +69,7 @@ object AccessType extends Enumeration {
               SHOWTABLES |
               SHOWFUNCTIONS |
               DESCFUNCTION => USE
-          case TRUNCATETABLE => UPDATE
+          case TRUNCATETABLE | OPTIMIZETABLE => UPDATE
```

Comment:
IIUC, OPTIMIZETABLE only changes data layout?

Reply:
Yes, what type should be used?

Reply:
It seems that MSCK is approaching.

Reply:
Alter access type? Does `VacuumCommand`  also is alter type?

Reply:
It depends on the impact of Vacuum on data or meta

Reply:
`ALTER` is used to modify metadata data of object, `UPDATE` is used to modify physical data of object. `OPTIMIZETABLE` merged the files to generate a new version. Although the data results have not changed, the data files have changed, so is UPDATE more appropriate?

Reply:
UPDATE is related to record changing

Reply:
I understand, change it immediately.